api/src/controllers/young/index.js

```diff
@@ -740,6 +740,12 @@ router.put("/:id/soft-delete", passport.authenticate("referent", { session: fals
     young.set({ status: YOUNG_STATUS.DELETED });
 
     await young.save({ fromUser: req.user });
+    const result = await patches.deletePatches(req, YoungObject);
```

Comment:
j'aime bien explicité les params de fonction perso, genre comme ca :
```suggestion
    const result = await patches.deletePatches({ user: req, model: YoungObject });
```

au moins on ne se trompe pas, c'est explicite. mais c'est plus du confort